policyengine_us/variables/gov/states/la/tax/income/exemptions/la_blind_or_disabled_exemption.py

```diff
@@ -0,0 +1,20 @@
+from policyengine_us.model_api import *
+
+
+class la_blind_or_disabled_exemption(Variable):
+    value_type = float
+    entity = TaxUnit
+    label = "Louisiana blind or disabled exemption"
+    unit = USD
+    definition_period = YEAR
+    reference = "https://www.legis.la.gov/legis/Law.aspx?d=102518"
+    defined_for = StateCode.LA
+
+    def formula(tax_unit, period, parameters):
+        amount = parameters(
+            period
+        ).gov.states.la.tax.income.exemptions.blind_or_disabled
+        blind_head = tax_unit("blind_head", period)
```

Comment:
![image](https://github.com/PolicyEngine/policyengine-us/assets/110687043/57452803-7929-42a6-86d4-bb4ebf172dd1)
Blind spouses get the credit

Reply:
so it should be blind_head or blind_spouse?

Reply:
it should be both